src/Uno.UI/UI/Xaml/Media/ImageSource.wasm.cs

```diff
@@ -19,6 +19,11 @@ namespace Microsoft.UI.Xaml.Media
 {
 	partial class ImageSource
 	{
+		public ImageSource()
+		{
+			InitializeBinder();
+		}
```

Comment:
I see it's pre-existing, but I think the `InitializeBinder` is not needed at all.